components/Modal/modal.css

```diff
@@ -0,0 +1,44 @@
+:root {
+  --border-color: rgba(0, 0, 0, 0.2);
+  --border-shadow: rgba(0 0 0 / 50%);
+}
```

Comment:
you can make a common stylesheet and add this in it

Reply:
Ok will do that